src/util/mask.js

```diff
@@ -90,7 +90,7 @@ const maskValidates = (mask, char) => {
  * @return {String}
  */
 export const maskText = (text, masked, dontFillMaskBlanks) => {
-  if (!masked.length || !text.length) return text
+  if (!masked.length || !text || !text.length) return text
```

Comment:
`if (!masked.length || !text) return text` should be enough

Reply:
Really? I'm already too deep into strongly typed language hahaa.

Thanks @jacekkarczmarczyk